### PR TITLE
Add setup.py and support for using as a module.

### PR DESCRIPTION
**Pull request by hlein**
Several proposed setup.py's exist in PRs for unoconv; this merges
and updates them.  Credits to @IuryAlves, @kellerkind, @shuge.

### CODE REVIEW

**Comment by hlein** [comment]
I noticed that some TravisCI checks failed, but looking at their details they appear to be errors unrelated to these changes.  Is there anything I can/should do to poke them, retry them, etc?

**Comment by strichter** [comment]
+1

**Comment by strichter** [comment]
@shuge I would love to see a new release on PyPI, since 0.7 has been officially released for a long time and fixes some important bugs.

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Always prefer setuptools over distutils
+from setuptools import setup, find_packages
+# To use a consistent encoding
+from codecs import open
+from os import path
+
+here = path.abspath(path.dirname(__file__))
+# Get the long description from the README file
+with open(path.join(here, 'README.adoc'), encoding='utf-8') as f:
+    long_description = f.read()
```
Comment (regebro) [inline setup.py]: You don't seem to use this in the setup()

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      # See https://pypi.python.org/pypi?%3Aaction=list_classifiers
+      classifiers = [
+        'Development Status :: 5 - Production/Stable',
+        'Environment :: Console',
+        'Intended Audience :: Developers',
+        'Intended Audience :: System Administrators',
+        'Intended Audience :: Education',
+        'License :: OSI Approved :: GNU General Public License v2 (GPLv2)',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
```
Comment (regebro) [inline setup.py]: I think we should specify supported Python versions, like 2.7, 3.5 etc.

**Comment by regebro** [comment]
Thanks! A setup.py has now been added, although not based on this one. That doesn't mean we don't appreciate the effort! Thanks again!

**Pull request closed without merging**